20180913/AGENDA.md

```diff
@@ -0,0 +1,21 @@
+# Sardana Follow-up Meeting - 2018/09/13
+
+To be held on 2018/09/13 at 14:00
+
+# Agenda
+
+1. Presentation
+    * Any volunteer?
+
+2. User Experience Report - Round Table
+
+3. Collaborative Developments - Review
+
+4. New Developments, Enhancements or Bug Fixes
+
+5. Pending PR distribution
+
+6. Any Other Business
+    * Sardana Documentation Camp summary.
+    * Brainstorming on how to improve and organize work.
```

Comment:
It would be nice if, before the meeting, all of us could think of few ideas on how to improve and organize our work.